source/io/PropertyNode.cpp

```diff
@@ -114,8 +112,8 @@ VIREO_FUNCTION_SIGNATUREV(PropertyNodeWrite, PropertyNodeWriteParamBlock)
     STACK_VAR(String, controlRefVINameVar);
     STACK_VAR(String, controlRefDataItemIdVar);
```

Comment:
Do we need to remove this variable? It appears it is not longer needed.

Reply:
You are right, good catch, I'll do it.